scripts/extensions/broadcasting/src/rundowns/rundown-view-edit.tsx

```diff
@@ -469,12 +467,13 @@ export class RundownViewEditComponent extends React.PureComponent<IProps, IState
                                                 },
                                             ]}
                                         >
-
-                                            <IconButton
-                                                ariaValue={gettext('Actions')}
-                                                icon="dots-vertical"
+                                            <button
```

Comment:
I thought we want to use components for buttons everywhere

Reply:
I know.... but I needed to edit the styles and add a class... but the IconBUtton component doesn't have a prop for className.. so I had to find a specific solution

Reply:
I get the requirements but not sure about the fix, we should then probably make those components if they are missing in the framework, that's pretty common usecase we have in other places as well

Reply:
thats one options.. another one is to add className prop on Button conponent in ui-framework side